src/marqo/tensor_search/configs.py

```diff
@@ -34,6 +34,7 @@ def default_env_vars() -> dict:
         EnvVars.MARQO_MAX_INDEX_FIELDS: None,
         EnvVars.MARQO_MAX_DOC_BYTES: 100000,
         EnvVars.MARQO_MAX_RETRIEVABLE_DOCS: 10000,
-        EnvVars.MARQO_MODELS_TO_PRELOAD: ['hf/all_datasets_v4_MiniLM-L6', "ViT-L/14"]
+        EnvVars.MARQO_MODELS_TO_PRELOAD: ['hf/all_datasets_v4_MiniLM-L6', "ViT-L/14"],
+        EnvVars.MARQO_LOG_LEVEL: "info"
```

Comment:
Can you add a comment next to `EnvVars.MARQO_LOG_LEVEL` saying:
`# this env var is set to info by default in run_marqo.sh, which overrides this value `

Reply:
done